src/net_processing.cpp

```diff
@@ -721,6 +721,26 @@ class PeerManagerImpl final : public PeerManager
     void UpdateChainTipSet(std::set<const CBlockIndex*>& chain_tips, const CBlockIndex* potential_new_tip)
         EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    /** Check if a block is on the main chain or best (more work than the active tip) chain. */
```

Comment:
unless `best` is some terminology, `better` should be more accurate?

Reply:
terminology i guess (`m_best_header`), feel free to ignore and close this comment